litmus-portal/frontend/src/components/Sections/Workflow/ChooseWorkflow/index.tsx

```diff
@@ -6,6 +6,7 @@ import useStyles, { CssTextField, ColorButton } from './styles';
 import useActions from '../../../../redux/actions';
 import * as WorkflowActions from '../../../../redux/actions/workflow';
 import PredifinedWorkflows from '../../../PredifinedWorkflows';
+import workflowsList from '../../../../models/data';
```

Comment:
Its a better choice to keep this in `preDefinedWorkflows/data.ts` models is to store the structure and types of the reusable data

Reply:
@S-ayanide, Yes now preDefinedWorkflows/data.ts is there. Any more to change?